.github/workflows/python-publish.yml

```diff
@@ -14,6 +14,24 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+      with:
+        persist-credentials: false
+        fetch-depth: 0
+    - name: Set version veriable
+      id: vars
+      run: echo ::set-env name=RELEASE_VERSION::${GITHUB_REF#refs/*/}
+    - name: Update versiona rel
+      run: sed -i -E "s/version=\\\"[0-9.]*\\\",/version=\\\"$RELEASE_VERSION\\\",/g" ./setup.py
```

Comment:
You can actually update the setup.py to have a `placeholder` for version and replace that here and then just build and deploy

Reply:
Answered here : https://github.com/paypay/paypayopa-sdk-python/pull/29#issuecomment-705312974